app/shared/profile-commands.js

```diff
@@ -14,6 +14,8 @@
 
 import * as profileCommandTypes from './constants/profile-command-types';
 
+import { ReadabilityResult } from '../shared/types';
```

Comment:
That should be `import type`, which is deliberately distinct from `import`, I imagine for that reason.